homeassistant/components/homematicip_cloud/climate.py

```diff
@@ -84,8 +85,50 @@ def current_humidity(self) -> int:
 
     @property
     def hvac_mode(self) -> str:
-        """Return current operation ie. automatic or manual."""
-        return HMIP_STATE_TO_HA.get(self._device.controlMode)
+        """Return hvac operation ie. heat, cool mode.
+
+        Need to be one of HVAC_MODE_*.
+        """
+        if self._device.boostMode:
+            return HVAC_MODE_HEAT
+
+        return HVAC_MODE_AUTO
```

Comment:
AUTO mean that you can't control the target temperature, so you should flip this

Reply:
or check self._device.manualMode or so

Reply:
OK, i switched it, and also added a check for manual mode.
It's possible to press boost mode in manual mode, but the AUTO should be displayed.